test/typecheck/main.go

```diff
@@ -197,6 +197,14 @@ func main() {
 
 	if len(args) == 0 {
 		args = append(args, "./...")
+		entries, err := os.ReadDir("./staging/src/k8s.io")
```

Comment:
heh, another reason this tool won't be fully general

trying to think if there's a way to avoid hardcoding this, maybe we walk and discover all go modules?

Reply:
(not to prioritize reuse, but to avoid missing anything)

Reply:
added a TODO, will hopefully get to it :)